usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -661,32 +654,10 @@ def load_file_c(submission_attributes, db_cursor, award_financial_frame):
             update_skipped_tas(row, skipped_tas)
             continue
 
-        # Find a matching transaction record, so we can use its subtier agency information to match to (or create) an
-        # Award record.
-
-        # Find the award that this award transaction belongs to. If it doesn't exist, create it.
-        filters = {}
```

Comment:
@nmonga91 If we have a way already to pass in options, like the `nosubaward` option, it may be better to just guard this block with another new option, leaving the code there to be optimized, but have jenkins run this stage with a `link_c_to_d_on_load` flag set to false.

If you were to guard this, the use of the sql linkage above would need to be activated by the same guard.

Reply:
The linkage should be attempted every time, I don't think we should allow the option to exclude it